rearrange/tasks.py

```diff
@@ -414,12 +375,17 @@ def drop_objects_and_teleport_back_them_to_original_locations(
 
                     if cur_pose["openness"] > 0.0:
                         self.unshuffle_env.controller.step(
-                            "CloseObject", objectId=object_id,
+                            "CloseObject",
+                            objectId=object_id,
+                            **self.unshuffle_env.physics_step_kwargs,
                         )
                     # NOTE: 'moveMagnitude' is 'openness' with the open object action.
                     # This is not a typo here.
                     self.unshuffle_env.controller.step(
-                        "OpenObject", objectId=object_id, moveMagnitude=goal_openness
+                        "OpenObject",
+                        objectId=object_id,
+                        moveMagnitude=goal_openness,
+                        **self.unshuffle_env.physics_step_kwargs,
                     )
                     action_success = self.unshuffle_env.last_event.metadata[
                         "lastActionSuccess"
```

Comment:
Any reason this is not using the new `OpenObject` with `openness`, instead the old one with `moveMagnitude`?

Reply:
No reason, fixed.